Echo.json5

```diff
@@ -65,7 +65,8 @@
     { name: 'AREnableMyCollectionHFOnboarding', value: false },
     { name: 'ARReorderSWAArtworkSubmissionFlow', value: true },
     { name: 'AREnableNewOpaqueImageComponent', value: false },
-    { name: 'ARIosSearchTabCuratedCollections', value: true },
+    { name: 'AREnableSearchDiscoveryContentIOS', value: true },
```

Comment:
I think we won't break anything by renaming `ARIosSearchTabCuratedCollections` to `AREnableSearchDiscoveryContentIOS`, since this flag is [not marked](https://github.com/artsy/eigen/blob/main/src/app/store/config/features.ts#L223) `readyForRelease` and the `echoFlagKey` is not specified in Eigen

Reply:
@dblandin JFYI instead of `AREnableSearchDiscoveryContentApple` I named `AREnableSearchDiscoveryContentIOS`